uint8 waypoint
uint8 upORdown